web/tests/settings_data.test.cjs

```diff
@@ -98,6 +102,26 @@ const students = {
     can_mention_group: 3,
     can_remove_members_group: 1,
     creator_id: 4,
+    deactivated: false,
+};
+const deactivated_students = {
+    description: "Students group",
+    name: "Students",
```

Comment:
User groups of same name are not allowed, so would be good to not have them in test data as well. Instead of creating a new group, the original students can also be set as deactivated for testing.